### PR TITLE
Rancher registry standalone package

### PR DESCRIPTION
**Pull request by izaac**
Package to create Docker private registries for use in Rancher.

- This package creates a docker registry with auth or without auth using valid certificates.
- The Rancher package was updated to use `CATTLE_SYSTEM_DEFAULT_REGISTRY` during helm install if a `registry_fqdn` variable exists in corral.

### CODE REVIEW

**Comment by izaac** [comment]
https://github.com/rancher/qa-tasks/issues/51

**Review by sbstp — APPROVED** [review]
Looks OK to me, but I'm still pretty new to corral.

**Comment by izaac** [comment]
The latest update just removes unused package variables from manifest.yaml

File: templates/standalone_registry/manifest.yaml
Comment (jakefhyde) [inline templates/standalone_registry/manifest.yaml:3]:
```suggestion
  A docker registry
```

**Review by jakefhyde — CHANGES_REQUESTED** [review]
Left some comments (mostly just nits). Also some general comments left on https://github.com/rancherlabs/corral-packages/pull/8 are also applicable here

File: templates/rancher/overlay/opt/corral/rancher/install-rancher.sh
Comment (igomez06) [reply templates/rancher/overlay/opt/corral/rancher/install-rancher.sh:26]: Formatting issues: https://github.com/rancherlabs/corral-packages/pull/8/files#r1161909034

File: templates/aws/registry_nodes/manifest.yaml
Comment (igomez06) [inline templates/aws/registry_nodes/manifest.yaml:55]: Will be cases with more than one registry node?

Comment (izaac) [reply templates/aws/registry_nodes/manifest.yaml:55]: I think this is a remnant from when I begun following the `agent_nodes`' design.
But then stepped back on that after I realized that's not the same case for registries.

I'll remove it.

Comment (izaac) [reply templates/aws/registry_nodes/manifest.yaml]: Removed unused variable
No re-test needed.

**Comment by izaac** [comment]
Hi @jakefhyde could I ask for another review when you're available?

File: templates/rancher/overlay/opt/corral/rancher/install-cert-manager.sh
Comment (jakefhyde) [inline templates/rancher/overlay/opt/corral/rancher/install-cert-manager.sh:33]: Formatting (indent)

File: templates/aws/registry_nodes/terraform/pools/main.tf
Comment (jakefhyde) [inline templates/aws/registry_nodes/terraform/pools/main.tf:16]:
```suggestion
provider "aws" {
  access_key = var.aws_access_key
  secret_key = var.aws_secret_key
  region     =  var.aws_region
}
```
Auto format file

**Review by jakefhyde — CHANGES_REQUESTED** [review]
Couple of nits

**Comment by izaac** [comment]
@jakefhyde I think I've addressed all the feedback from the last round of review. Could I have another pass?

**Review by igomez06 — APPROVED** [review]
Looks good at least on my end. Will wait for Jake's final approval.

**Review by jakefhyde — APPROVED** [review]

**Comment by igomez06** [comment]
@jakefhyde could you merge the PR, I'll rebase the changes into branch. Thank you!

**Comment by izaac** [comment]
This last change added a missing variable in the `registries.yaml` package manifest.

**Comment by izaac** [comment]
Are we good for merge here ?

**Comment by izaac** [comment]
Hi @jakefhyde please check the PR. It is approved.

**Pull request merged**
